docs/source/installation.mdx

```diff
@@ -120,25 +120,3 @@ git pull
 ```
 
 Your Python environment will find the `main` version of 🤗 Diffusers on the next run.
-
-## Notice on telemetry logging
-
-Our library gathers telemetry information during `from_pretrained()` requests.
```

Comment:
We still send telemetry when the user calls `from_pretrained(...)` simple because otherwise it's impossible to download the weights - so I think it would be better to keep this. @anton-l what do you think?

Reply:
why is it impossible to download the weights without telemetry? I am able to access all the same data with simple wget requests